src/history/style.ts

```diff
@@ -9,7 +9,6 @@ export const HistoryItem = styled.View`
   padding: 12px 16px;
   border-bottom-width: 0.25px;
   border-bottom-color: ${({ theme }) => theme.palette.neutral['200']};
-  height: ${HISTORY_ITEM_HEIGHT}px;
```

Comment:
If this line is removed, then maybe the HISTORY_ITEM_HEIGHT should be removed as well 🤔

Reply:
I kept that to give the flashlist 'general' height, which improves performance